### ISSUE TITLE
Normalise Font Sizes

### ISSUE THREAD

**Issue by PBI-David** [microsoft/charticulator#607]
**Describe the bug**
Same font and size renders differently in Charticulator and PBI Desktop

**To Reproduce**
1. Create a table with Tahoma size 10 values in PBI
2. Create a chart legend with Tahoma size 10 in Charticulator
3. View side by side

**Expected behavior**
Font sizes should be the same 

**Screenshots**
Both fonts below are Tahoma size 10
![Example](https://i.imgur.com/rpHYg2D.png)

**Diagnostic information**
N/A

**Additional context**
N/A

**Comment by zBritva** [comment]
https://github.com/microsoft/charticulator/issues/548

**Comment by PBI-David** [comment]
Thanks. Is this definitely a resize issue though as the charticulator visual was expanded to its maximum size and I would expect two identical fonts to render at the same size.

**Comment by zBritva** [comment]
Try to use workaround from https://github.com/microsoft/charticulator/issues/584#issuecomment-865801531 if chart size and visual size is same, font size should be identical.

**Comment by PBI-David** [comment]
I've uploaded a pbix so you can see they definitely render differently. Unless I'm misunderstanding, I don't think workaround in #584 applies as this is simply a legend and I would expect font sizes to be identical between the two.  
[Font.zip](https://github.com/microsoft/charticulator/files/6783371/Font.zip)

**Comment by zBritva** [comment]
yes, I see. The problem is related to SVG scale. The core visual renders text in HTML, Charticulator uses SVG and renders text in SVG.
On scale the SVG document, texts change scales too.  The visual has wrong default scale.

**Comment by PBI-David** [comment]
Thanks. Is this something an end user can configure or is a code change required?

**Comment by zBritva** [comment]
Code changing is required.
